README.md

```diff
@@ -25,8 +25,8 @@ var mediator = require('fh-wfm-mediator');
 var Topics = require('fh-wfm-mediator/lib/topics');
 
 var topics = new Topics(mediator)
-  .prefix('wfm')
-  .entity('user')
+  .withPrefix('wfm')
+  .withEntity('user')
```

Comment:
Ah yeah, that's what happens when I'm lazy and just look at the the comment below the code, and not the full file :)

Still, having some linter that looks at js script tags in your .md files would be pretty awesome! Add that one to the laundry list of "projects I plan to do but realistically will never get to".